insights/parsers/system_time.py

```diff
@@ -103,7 +103,30 @@ class NTP_conf(NTPConfParser):
 
     Uses the ``NTPConfParser`` class defined in this module.
     """
-    pass
+    def get_tinker(self, param):
```

Comment:
What about just adding a second arg so that it could be called as `get_last('tinker', 'step')`?  That would seem to make it more generally useful for all parameters here.

Reply:
OK, I've redone this as a `get_param` method and renamed my branch.  I think I've pushed it to this merge request - let's see...